site/content/entry/citra-progress-report-2023-q1/index.md

```diff
@@ -0,0 +1,409 @@
+
+NOTE: This PR does not mean that Citra Android has multiplayer access. This is only related to homebrew apps at the moment.
+
+### Allow opening in-emulation menu by pressing Back ([#6248](https://github.com/citra-emu/citra/pull/6248)) by [JosJuice](https://github.com/JosJuice)
+
+With the Android 13 update released on some devices came a big issue with Citra. No longer could you swipe down from the top to open the in-game menu! Instead, just the phone notification bar would be brought down.
+
+Due to this, you could no longer access Save States, Amiibo, Screen Layout, Overlay, FPS counter and so many more vital options to use Citra! This was a massive problem!
+
+The solution was to just use the back button found on all devices to open the in-emulation menu instead! This can be accessed from the bottom of the device, next to the home button and recents button. By doing this, the in-emulation menu can be accessible on a number of devices now, such as Chromebooks running Citra Android and VR headsets, not just devices running Android 13.
+
+# Emulation Accuracy
+
+### Implement APT command 0x0103 ([#5478](https://github.com/citra-emu/citra/pull/5478)) by [xperia64](https://github.com/xperia64)
+
+This PR allows for controls introduced by the New Nintendo 3DS to be utilized in Citra whilst **New 3DS mode** is enabled. The New Nintendo 3DS came with the addition of the C-Stick, and the ZL and ZR trigger buttons. 
+These can now be used in games such as Super Smash Bros. for Nintendo 3DS to give you more control over how you play your game!
+
+### Initial implementation of partial_embedded_buffer_dirty handling ([#5548](https://github.com/citra-emu/citra/pull/5548)) by [xperia64](https://github.com/xperia64)
+
+For a while, the Mii voices in Tomodachi Life would continuously cut off before they would finish speaking, which could make it hard to understand what they are saying to the player. By implementing a partial embedded buffer, a buffer being an area of memory which stores data on a temporary basis, you can hear the sweet voices of your favorite Miis again!
+
+This also fixes an issue where FMVs in Detective Pikachu would run faster than what the frame rate allowed for.
+
+{{< figure src="tomodachi.png"
+    title="Make sure to take your hayfever medicine, Frank!" >}}
+
+### service/apt: Implement Deliver Args ([#5611](https://github.com/citra-emu/citra/pull/5611)) by [zhaowenlan1779](https://github.com/zhaowenlan1779)
+
+This partially fixes an issue where some games would just show black and white screens when launched. By implementing Deliver Args, a way to pass arguments from the caller to the callee, as well as `PrepareToDoApplicationJump` and `JumpToApplication` APT services. This allows the game to crash with a fatal error instead of just hanging on a black screen. 
```

Comment:
```suggestion
This partially fixes an issue where some games would just show black and white screens when launched. By implementing Deliver Args, a way to pass arguments from one 3DS application to another, as well as `PrepareToDoApplicationJump` and `JumpToApplication` APT services. It allows the game to crash with a fatal error instead of just hanging on a black screen. 
```